addon/-private/create-needle-haystack-helper.js

```diff
@@ -18,16 +18,18 @@ const { K } = Ember;
  */
 export default function createNeedleHaystackHelper(fn = K) {
   return Helper.extend({
-    content: computed('needle.[]', 'haystack.[]', function() {
+    content: computed('needle.[]', 'haystack.[]', 'option', function() {
       let needle = get(this, 'needle');
       let haystack = get(this, 'haystack');
+      let option = get(this, 'option');
 
-      return fn(needle, haystack);
+      return fn(needle, haystack, option);
     }).readOnly(),
 
-    compute([needle, haystack]) {
+    compute([needle, haystack, option]) {
```

Comment:
I think `option` needs to go before `haystack`. The subject where the helper is operating on should always be last.